ext/xvalidator/validator.go

```diff
@@ -67,7 +67,15 @@ func NewWithPrefix(prefix string) (*validator.Validate, ut.Translator) {
 
 	validate.RegisterValidation("accaddress", IsAccAddress)
 	validate.RegisterTranslation("accaddress", trans, func(ut ut.Translator) error {
-		return ut.Add("hash", "{0} must be a valid address", false)
+		return ut.Add("address", "{0} must be a valid address", false)
+	}, func(ut ut.Translator, fe validator.FieldError) string {
+		t, _ := ut.T(fe.Tag(), fe.Field(), prefix)
+		return t
+	})
+
+	validate.RegisterValidation("coins", IsCoins)
```

Comment:
not needed if we use the coin type

Reply:
I think this will style be useful to check the denom and the amount is correctly formated. but it need to be updated to use directly the `Coins` type.
Or does the marshalling function of `Coins` already check the format?